examples/tests/sdk_tests/function/invoke_async.test.w

```diff
@@ -0,0 +1,19 @@
+bring cloud;
+bring util;
+
+let payload = "hello";
+log("log preflight");
+
+let f1 = new cloud.Function(inflight (input: str): str => {
+  log("log inside f1");
+}) as "f1";
+
+let f2 = new cloud.Function(inflight (input: str): str => {
+  log("log inside f2");
+}) as "f2";
+
+test "invoke" {
+  log("log inside test");
+  f1.invokeAsync(payload);
```

Comment:
Looks like there's an issue with test flakeyness (see https://github.com/winglang/wing/pull/4434/commits/ad97a8a2655cae868735fbaf43ab2b55dc0a09e4). I believe the issue is f1's code might not get run until after the test finished, but sometimes it will. If we want to guarantee "log inside f1" is printed in the test output, we should add something like `util.sleep(1s)` to the end of the test